<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsDialog</class>
 <widget class="QDialog" name="AccountsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accounts Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="accountsList"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="remoteBox">
     <property name="title">
      <string>Remote Host</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Hostname</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="remoteHostname"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>API Secret</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="remoteSecret"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="remotePort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="accountBox">
     <property name="title">
      <string>Remote Account</string>
     </property>
     <widget class="QLineEdit" name="remotePassword">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>125</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="remoteUsername">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>28</y>
        <width>125</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>28</y>
        <width>64</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>59</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="activeBox">
     <property name="title">
      <string>Active</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="activeRelLabel">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="activeRel">
        <item>
         <property name="text">
          <string>Text (no link)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Link</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Audio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="activeTextLabel">
        <property name="text">
         <string>Display Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="activeText"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="activeLinkUrl">
        <property name="text">
         <string>Url</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="activeUrl"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="activeLinkPortPath">
        <property name="text">
         <string>Port + Path</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="activePort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="activeLinkPath">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="activePath"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="activeIpMine">
        <property name="toolTip">
         <string>For the port+path and path options</string>
        </property>
        <property name="text">
         <string>Use my ip address, not remote's</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="inactiveBox">
     <property name="title">
      <string>Inactive</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="inactiveRelLabel">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="inactiveRel">
        <item>
         <property name="text">
          <string>Text (no link)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Link</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Audio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="inactiveTextLabel">
        <property name="text">
         <string>Display Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inactiveText"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="inactiveLinkUrl">
        <property name="text">
         <string>Url</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="inactiveUrl"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="inactiveLinkPortPath">
        <property name="text">
         <string>Port + Path</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="inactivePort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="inactiveLinkPath">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="inactivePath"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="inactiveIpMine">
        <property name="toolTip">
         <string>For the port+path and path options</string>
        </property>
        <property name="text">
         <string>Use my ip address, not remote's</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="accountsAdd">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="accountsDel">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
